sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -946,83 +666,30 @@ class RocksDBSuite extends AlsoTestWithChangelogCheckpointingEnabled with Shared
       dfsRootDir.getAbsolutePath, Utils.createTempDir(), new Configuration)
     val changelogWriter = fileManager.getChangeLogWriter(1, true)
     (1 to 5).foreach { i =>
-      changelogWriter.put(i.toString, i.toString, StateStore.DEFAULT_COL_FAMILY_NAME)
+      changelogWriter.put(i.toString, i.toString)
     }
     (1 to 5).foreach { i =>
-      changelogWriter.merge(i.toString, i.toString, StateStore.DEFAULT_COL_FAMILY_NAME)
+      changelogWriter.merge(i.toString, i.toString)
     }
 
     (2 to 4).foreach { j =>
-      changelogWriter.delete(j.toString, StateStore.DEFAULT_COL_FAMILY_NAME)
+      changelogWriter.delete(j.toString)
     }
 
     changelogWriter.commit()
     val changelogReader = fileManager.getChangelogReader(1, true)
     val entries = changelogReader.toSeq
     val expectedEntries = (1 to 5).map { i =>
-      (RecordType.PUT_RECORD, i.toString.getBytes,
-        i.toString.getBytes, StateStore.DEFAULT_COL_FAMILY_NAME)
+      (RecordType.PUT_RECORD, i.toString.getBytes, i.toString.getBytes)
     } ++ (1 to 5).map { i =>
-      (RecordType.MERGE_RECORD, i.toString.getBytes,
-        i.toString.getBytes, StateStore.DEFAULT_COL_FAMILY_NAME)
+      (RecordType.MERGE_RECORD, i.toString.getBytes, i.toString.getBytes)
     } ++ (2 to 4).map { j =>
-      (RecordType.DELETE_RECORD, j.toString.getBytes,
-        null, StateStore.DEFAULT_COL_FAMILY_NAME)
+      (RecordType.DELETE_RECORD, j.toString.getBytes, null)
     }
 
     assert(entries.size == expectedEntries.size)
     entries.zip(expectedEntries).map{
-      case (e1, e2) => assert(e1._1 === e2._1 && e1._2 === e2._2 &&
-        e1._3 === e2._3 && e1._4 === e2._4)
-    }
-  }
-
-  testWithChangelogCheckpointingEnabled(
```

Comment:
This suite is removed because column family is no longer part of the input parameter in changelog v2.